keanacobarde/HHPWsBe
# FILE SETUP + CLASS CREATION

## Description
- All files required to exist, exist. Folder hierarchy is structured.
- Created DbContext.
- Removed boilerplate code.
- Created APIs, test data, and models folder
- Created class structure

## Related Issue
#2 

## Motivation and Context
- This setup is required for the 

## How Can This Be Tested?
- Pull PR code to local testing repo
- Ensure that all created folders include files
- Ensure DbContext is structured correctly

## Screenshots (if appropriate):
N / A

## Types of changes
